draft-packages/form/KaizenDraft/Form/Primitives/InputSearch/InputSearch.tsx

```diff
@@ -0,0 +1,97 @@
+import { Icon } from "@kaizen/component-library"
+import { LoadingSpinner } from "@kaizen/draft-loading-spinner"
+import classnames from "classnames"
+import React from "react"
+import search from "@kaizen/component-library/icons/search.icon.svg"
+import clear from "@kaizen/component-library/icons/clear.icon.svg"
+
+import styles from "../Input/styles.scss"
+
+export interface InputSearchProps
+  extends Omit<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    "className" | "defaultValue"
+  > {
+  classNameAndIHaveSpokenToDST?: string
+  reversed?: boolean
+  loading?: boolean
+  secondary?: boolean
+  onClear?: () => void
+}
+
+const InputSearch: React.FunctionComponent<InputSearchProps> = (
+  props: InputSearchProps
```

Comment:
/meganit, this redeclaration of `props: InputSearchProps` is redundant as you've defined the prop types already in the line above.

Reply:
Wait, really? Setting it on the return type makes this obsolete?

Reply:
`React.FunctionComponent` accepts a generic which is used to type props, so Ally is correct. It's not needed again.